src/Microsoft.Azure.SignalR/ServiceRouteBuilder.cs

```diff
@@ -18,6 +21,7 @@ public class ServiceRouteBuilder
     {
         private readonly IServiceProvider _serviceProvider;
         private readonly RouteBuilder _routes;
+        private readonly IList<Func<TimeSpan, Task>> _shutdownHooks = new List<Func<TimeSpan, Task>>();
```

Comment:
nit: Use the concrete List type instead of the interface for private/internal properties